medusa/helpers/__init__.py

```diff
@@ -1574,11 +1533,13 @@ def get_disk_space_usage(disk_path=None, pretty=True):
 
 def get_tvdb_from_id(indexer_id, indexer):
 
-    session = make_session()
+    content_session = ContentSession()
+    session = MedusaSession()
```

Comment:
But I'd like to have some sugar coating, to be able to easily 'drop-in', where we expect content, or json. Should I go with the get_json() then?